Dependencies/xr/Source/OpenXR/XrRegistry.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include "XrSupportedExtensions.h"
+
+#ifndef XR_REGISTRY
+#define XR_REGISTRY

```

Comment:
nit: Is this idiom not redundant on `#pragma once`?

Reply:
this should now be fixed